src/pages/iou/ReceiptSelector/NavigationAwareCamera.js

```diff
@@ -21,10 +28,56 @@ const defaultProps = {
     torchOn: false,
 };
 
+function useTabNavigatorFocus({cameraTabIndex, isInTabNavigator}) {
```

Comment:
I would suggest moving this to its own file and having it be defined as a proper hook.

Reply:
I didn't request this change because there's already separate GH for this.
- https://github.com/Expensify/App/issues/29527

Reply:
agree!